examples/tensat.lisp

```diff
@@ -158,7 +172,16 @@
       (cfg-blocks 3 2 2 1024))
     tmp))
 
-#+nil (let* ((*egraph* (make-egraph :analyses 'shape))
+(let* ((*egraph* (make-egraph :analyses '(shape cost)))
        (a (resnext-50)))
   (egraph-rebuild)
-  (get-analysis-data a 'shape))
+  (get-analysis-data a 'cost))
+
+(let* ((*egraph* (make-egraph :analyses '(shape cost)))
+      (input (list 'input 2 3))
+      (weight (list 'weight 3 5))
+      (add (make-term (list 'ewmul input input)))
+      (matmul (make-term (list 'matmul input weight))))
+  (egraph-rebuild)
+  (assert (= 6 (get-analysis-data add 'cost)))
+  (assert (= 30 (get-analysis-data matmul 'cost))))
```

Comment:
For now let's add a `#+nil` before this whole top level form so it doesn't run when we compile/load the file.

At some point we should make this a `def-test` (and replace `assert` with `fiveam:is`).